### PR TITLE
Travis SCSS review

### LINKED ISSUE #34: Add CSS duplication prevention to Travis
http://zmoazeni.github.io/csscss/

### PR DESCRIPTION
**Pull request by AronNovak**
## Status

integrated tools as they work on local

## Todo

make it work on Travis (with potentially a dumb SCSS file to verify)

will fix #34

### CODE REVIEW

**Comment by AronNovak** [comment]
image

The linting works through Travis, now making it green.

File: ci-scripts/install_scss.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Install SCSS CI check dependencies.
+#
+# ---------------------------------------------------------------------------- #
+
+# Check the current build.
+if [ -z "${SCSS_REVIEW+x}" ] || [ "$SCSS_REVIEW" -ne 1 ]; then
+ exit 0;
+fi
+
+npm install -g stylelint
```
Comment (amitaibu) [inline ci-scripts/install_scss.sh]: maybe lock the version?

Comment (AronNovak) [inline ci-scripts/install_scss.sh]: Absolutely!

**Comment by amitaibu** [comment]
I think it also needs an entry in the README, and possible `npm lint` (?) or a bash script to be able to easily execute it locally.

**Comment by AronNovak** [comment]
CI-side check works sane again:
image

**Comment by AronNovak** [comment]
And the successful outcome:
image

https://travis-ci.org/Gizra/drupal-elm-starter/jobs/273463601

**Pull request closed without merging**